course_catalog/tasks.py

```diff
@@ -35,16 +36,26 @@ def get_edx_data(force_overwrite=False):
         settings.EDX_API_URL
         and settings.EDX_API_CLIENT_ID
         and settings.EDX_API_CLIENT_SECRET
+        and settings.OCW_LEARNING_COURSE_BUCKET_NAME
+        and settings.OCW_LEARNING_COURSE_ACCESS_KEY
+        and settings.OCW_LEARNING_COURSE_SECRET_ACCESS_KEY
     ):
         log.warning("Required settings missing for get_edx_data")
         return
     url = settings.EDX_API_URL
     access_token = get_access_token()
 
+    edx_data = {
+        "count": None,
```

Comment:
Why is `count` set to `None`?  Doesn't seem like it ever gets updated to another value.

Reply:
No particular reason other than lack of other sensible defaults. It's populated to be an `int` later. We can change it to be `0` if you like.

Reply:
I think you might be able to leave it out here entirely, and it will get included later when it's assigned the count number.

Reply:
My opinion is that it may be clearer for the key to exist with some default value in the initial dictionary declaration such that I can glance at the dictionary declaration and expect what keys are there; of course, dictionaries are mutable, so it's not exactly a promise on content schema, but it can serve as a useful signpost for someone skimming the code. 

I'll remove it, unless you change your mind. Ultimately, it's your call.

Reply:
ok, how about leaving it in there but initializing it to 0

Reply:
:+1: